### ISSUE TITLE
Factor out the modal component from the Query builder and create a wrapper for it.

### ISSUE THREAD

**Issue by ddelpiano** [openworm/geppetto-client#43]

**Comment by ddelpiano** [comment]
@filippomc this might be interesting for you as well for the control panel, Matteo suggested to mention you, we can have a call at some point this week about this.

**Comment by filippomc** [comment]
Yes it seems to be the same approach we are following for the new control panel. The control panel will be a collection of list viewers each one with its filter and pagination.
